variables.tf

```diff
@@ -425,3 +425,9 @@ variable "custom_alarms" {
   default     = {}
   description = "Map of custom CloudWatch alarms configurations"
 }
+
+variable "use_name_prefix" {
+  type        = bool
+  default     = true
```

Comment:
should be `false` to kept the current behaviour

Reply:
Actually the current behavior uses the name prefix...